scripts/gobuild.sh

```diff
@@ -16,16 +16,17 @@ set -x
 set -e
 export TOPWD="$(pwd)"
 export BUILDDIR="$(mktemp -d)"
-export GOPATH="${TOPWD}/ecs-init/:${BUILDDIR}"
 export SRCPATH="${BUILDDIR}/src/github.com/aws/amazon-ecs-agent"
+export GOPATH="${TOPWD}:${BUILDDIR}"
+export GO111MODULE="auto"
```

Comment:
Why this change?

Reply:
These changes were pulled from the `scripts/gobuild.sh` from `dev` to fix a broken build. Since we keep a different AL1 (legacy) and AL2 branches, I pulled these changes in.